dev/com.ibm.ws.microprofile.metrics.cdi.3.0/bnd.bnd

```diff
@@ -0,0 +1,68 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+Bundle-Name: com.ibm.ws.microprofile.metrics.cdi.3.0
+Bundle-SymbolicName: com.ibm.ws.microprofile.metrics.cdi.3.0
+Bundle-Description: MicroProfile Metrics CDI Integration, version ${bVersion}
+
+Import-Package: \
+	javax.enterprise.*;version="[1.1,3)",\
+	*
+
+Export-Package: \
+	io.astefanutti.metrics.cdi30,\
+	com.ibm.ws.microprofile.metrics.cdi.decorator,\
+	com.ibm.ws.microprofile.metrics.cdi.binding,\
+	com.ibm.ws.microprofile.metrics.cdi30.interceptors,\
+	com.ibm.ws.microprofile.metrics.cdi30.producer,\
+	com.ibm.ws.microprofile.metrics.cdi30.helper
+
+Private-Package: \
+	com.ibm.ws.microprofile.metrics.cdi.resources,\
+	com.ibm.ws.microprofile.metrics.cdi30.helper
+
+Include-Resource: \
+    META-INF=resources/META-INF
+
+WS-TraceGroup: METRICS
+
+-cdiannotations:
+
+-dsannotations-inherit: true
+-dsannotations: \
+	io.astefanutti.metrics.cdi30.MetricsExtension
+
+
+-buildpath: \
+	com.ibm.ws.logging;version=latest,\
+	com.ibm.ws.adaptable.module;version=latest,\
+	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
```

Comment:
Should cdi be 2.0?

Reply:
Fixed in the earlier https://github.com/OpenLiberty/open-liberty/pull/12299/commits/6b39b62bde8dcd27d91dad471d48ed340aca470f